### PR TITLE
Awesome books with ES6

### PR DESCRIPTION
**Pull request by techmoves**
in this repo i did 👍 👍 

- [x] No linters error
- [x] proper git hub work flow
- [x] Used ES 6
- [x] date display under nav bar

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+// Function to update the live time
+function updateLiveTime() {
+  // Get the current time using Luxon
+  const currentTime = DateTime.local();
+
+  // Format the current time as desired (e.g., 'HH:mm:ss')
+  const formattedTime = currentTime.toFormat('HH:mm:ss');
+  // Update the live-time element with the formatted time
+  liveTimeElement.textContent = formattedTime;
+}
```
Comment (Shisui6) [inline index.js]: Kindly update this function to use the es6 arrow syntax to adhere to the project requirements🙏

File: luxon.js
Comment (Shisui6) [inline luxon.js:1]: Kindly remove unused files from your project to keep your codebase clean👍

File: README.md [hunk 2]
```diff
@@ -1 +1,195 @@
...
+(back to top)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
```
Comment (Shisui6) [inline README.md]: Kindly add a message acknowledging whoever inspired this project🙏

**Review by Shisui6 — CHANGES_REQUESTED** [review]
Hi @techmoves, 

# Requires Changes 🍀
You have done a great job so far!👏👏
Your design is good 🖼️, and your code follows best practices. However, there are a few issues that you still need to work on🪲.

## Highlights
- Nice use of ES6 syntax ✅
- App works as expected ✅


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: luxon.js
Comment (techmoves) [reply luxon.js:1]: resolved

File: README.md (hunk 2 above)
Comment (techmoves) [reply README.md]: resolved

**Review by adel-gu — APPROVED** [review]
# Status: APPROVED 🟢 

**Hi @techmoves  👋🏼** 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Es6 syntax is well used ✔️ 
- Acknowledge section is updated ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**